src/monopoly/board/boardConstants.js

```diff
@@ -47,4 +47,12 @@ global.BUY_PROMPT = 2;
 global.BUY_ANSWER = 3;
 global.POST_TURN = 4;
 global.POST_TURN_ANSWER = 5;
-global.ENDED_TURN = 6;
+global.PROPOSE_TRADE = 6;
+global.TRADE_ANSWER = 7;
+global.ENDED_TURN = 8;
+
+//btnPressed
+global.BTN1 = 0;
```

Comment:
maybe at least a comment to explain the different buttons. Or better variable names?